src/identifier.ts

```diff
@@ -74,6 +74,8 @@ export const IdentifierType = makeEnum({
   StreamUserId: 'streamUserId',
   /** A token used to make API calls on behalf of a Plaid account. */
   PlaidProcessorToken: 'plaidProcessorToken',
+  /** An ID for an applicant on Onfido */
+  OnfidoApplicantId: 'onfidoApplicantId',
```

Comment:
I wonder if we should have a integration specific id that is more general. Cc @michaelfarrell76

Reply:
Can merge this for now